workspaces/editors.md

```diff
@@ -178,3 +179,38 @@ participants. If they do, click **Accept** to proceed.
 
 At this point, you'll be able to share your project and work with your partner
 in real-time.
+
+## RStudio
+
+Coder supports [RStudio](rstudio.com). To create a workspace that lets you use
+RStudio:
+
+1. Create an image that includes RStudio (see
+   [rocker/rstudio](https://hub.docker.com/r/rocker/rstudio) for an example).
```

Comment:
I'm sort of in agreement. `rocker/rstudio` _seems_ to be the image that most RStudio people use, but it does a lot of stuff that isn't necessary for just running rstudio.

Also fun fact, `rocker/rstudio` will error on startup if a password environment variable isn't set (https://github.com/rocker-org/rocker-versioned2/blob/3456299fbbd1190a0aefab7ad6e0a3239932a0b4/scripts/userconf.sh#L26-L35). However, when this image is used in Coder, it doesn't error because the entrypoint for the image is overwritten, meaning that this script doesn't get ran.

Reply:
We should - and it will have to have a known `passwd` for the user

Reply:
We should make sure these requirements are listed as well:
- `pgrep` and `rserver` must be in the shell's PATH.
- `rserver` must be configured to start on port 8787 (the default).